visualization/app/codeCharta/ui/codeMap/threeViewer/threeSceneService.ts

```diff
@@ -58,6 +60,11 @@ export class ThreeSceneService implements CodeMapPreRenderServiceSubscriber {
 		this.scene.add(this.lights)
 	}
 
+	onMapColorsChanged(mapColors: MapColors) {
+		this.folderLabelColorSelected = mapColors.selected
+		this.numberOrangeColor = ColorConverter.convertHexToNumber(this.folderLabelColorSelected)
```

Comment:
Is it still orange => `this.numberOrangeColor`?

Reply:
I renamed all occurrences